
# HG changeset patch
# User Hans de Goede <hdegoede@redhat.com>
# Date 1265703124 -3600
# Node ID 2b47f55000a5576c6ab20636ea41bdd3a6786387
# Parent  ab2eec3754f562ba08bf5ec65145a9cd15244009
libv4l: update upside down device table

From: Hans de Goede <hdegoede@redhat.com>

libv4l: update upside down device table

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Mon Feb 08 09:58:42 2010 +0100
+++ b/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Tue Feb 09 09:12:04 2010 +0100
@@ -245,6 +245,8 @@
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x13d3, 0x5094, 0, "ASUS CORPORATION            ", "K50C      ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
+  { 0x13d3, 0x5094, 0, "ASUSTeK Computer Inc.        ", "K50IJ     ",
+    V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x13d3, 0x5094, 0, "ASUSTeK Computer Inc.        ", "P50IJ     ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x174f, 0x1408, 0, "ASUSTeK Computer Inc.        ", "K40IN      ",

